tests/basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/server/connections/connection/factory/ConnectionFactoryTest.java

```diff
@@ -92,7 +82,15 @@ public void connectionFactoryUpdate() throws Exception {
         FormFragment form = page.getConnectionFactoryForm();
         table.bind(form);
         table.select(CONN_FAC_UPDATE);
-        crudOperations.update(connectionFactoryAddress(SRV_UPDATE, CONN_FAC_UPDATE), form, CALL_TIMEOUT, 123L);
+        crudOperations.update(connectionFactoryAddress(SRV_UPDATE, CONN_FAC_UPDATE), form,
+                formFragment -> {
+                    formFragment.number(CALL_TIMEOUT, 123L);
+                    formFragment.flip("use-topology-for-load-balancing", false);
```

Comment:
Since `use-topology-for-load-balancing` is a constant, I'd prefer having it in the `MessagingFixtures` class